app/src/main/res/values/strings.xml

```diff
@@ -138,9 +138,9 @@
     <string name="play_btn_text">Play</string>
     <string name="content">Content</string>
     <string name="show_age_restricted_content_title">Age restricted content</string>
-    <string name="video_is_age_restricted">Show age restricted video. Allowing such material is possible from Settings.</string>
-    <string name="duration_live">LIVE</string>
-    <string name="duration_live_button" translatable="false">LIVE</string>
+    <string name="video_is_age_restricted">Show age restricted video. Future changes are possible from \"Settings\".</string>
+    <string name="duration_live">Live</string>
+    <string name="duration_live_button" translatable="false">Live</string>
```

Comment:
We should check why there are two strings with the same value and why one is not translatable.
Apart from that, we need to modify the .xml files to force capital letters.

Reply:
Checked the usages and those changes are fine, because `android:textAllCaps="true"` is already used in XML resource files.